finagle-http/src/main/scala/com/twitter/finagle/http/filter/StatsFilter.scala

```diff
@@ -1,9 +1,29 @@
 package com.twitter.finagle.http.filter
 
-import com.twitter.finagle.{Service, SimpleFilter}
-import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.finagle._
 import com.twitter.finagle.http.{Request, Response, Status}
-import com.twitter.util.{Duration, Future, Return, Stopwatch, Throw}
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.util._
+
+object StatsFilter {
+  val role: Stack.Role = Stack.Role("HttpStatsFilter")
+  val description: String = "HTTP Stats"
+
+  def module[Req <: Request]: Stackable[ServiceFactory[Req, Response]] =
```

Comment:
I still don't think you need this type-param.

Please, note that it has nothing to do with your custom HTTP types (either for request or response) since at that point they should be lifted to something Finagle can work with (i.e., `c.t.f.http.Request` and `c.t.f.http.Response`).

By the same reason, `StatsFilter` doesn't need a type param `Req`, but we can't remove it yet (an API break) so let's keep it (filter) as is, but remove the type-param on a module.